src/main/java/org/jenkins/plugins/lockableresources/queue/LockRunListener.java

```diff
@@ -36,7 +36,7 @@ public class LockRunListener extends RunListener<Run<?, ?>> {
     public void onStarted(Run<?, ?> build, TaskListener listener) {
         // Skip locking for multiple configuration projects,
         // only the child jobs will actually lock resources.
-        if (build instanceof MatrixBuild) return;
+        if (build.getClass().getName().equals("hudson.matrix.MatrixBuild")) return;
```

Comment:
Missed https://github.com/jenkinsci/lockable-resources-plugin/blob/487e7db2bcde329d8417c95c023982e9dcb7363c/src/main/java/org/jenkins/plugins/lockableresources/queue/LockRunListener.java#L99 and https://github.com/jenkinsci/lockable-resources-plugin/blob/487e7db2bcde329d8417c95c023982e9dcb7363c/src/main/java/org/jenkins/plugins/lockableresources/queue/LockRunListener.java#L108 and https://github.com/jenkinsci/lockable-resources-plugin/blob/487e7db2bcde329d8417c95c023982e9dcb7363c/src/main/java/org/jenkins/plugins/lockableresources/queue/LockRunListener.java#L13